plugin/providers/phpipam/resource_phpipam_subnet.go

```diff
@@ -3,7 +3,9 @@ package phpipam
 import (
 	"errors"
 	"fmt"
+	"github.com/pavel-z1/phpipam-sdk-go/phpipam"
```

Comment:
I guess that in here you can not indicate the dependence from phpipam-sdk-go.
Try to remove this line please.